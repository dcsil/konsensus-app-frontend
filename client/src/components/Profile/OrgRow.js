import React from 'react';

import {
  DropdownMenu,
  DropdownItem,
  UncontrolledDropdown,
  DropdownToggle,
  UncontrolledTooltip,
} from 'reactstrap';
import Avatar from './Avatar';
// core components

const OrgRow = ({ user }) => {
  return (
    <tr>
      <td>
        <div className="avatar-group">
          <a
            className="avatar avatar-sm"
            href="#org-row"
            id="tooltip742438047"
            onClick={(e) => e.preventDefault()}
          >
            {user && (
              <Avatar
                name={`${user.firstName} ${user.lastName}`}
                url={user.image}
              />
            )}
          </a>
          <UncontrolledTooltip delay={0} target="tooltip742438047">
            {user && user.email}
          </UncontrolledTooltip>
        </div>
      </td>
      <td>
        <span className="mb-0 text-sm">
          {user && user.firstName} {user && user.lastName}
        </span>
      </td>
      <td>{user && user.role}</td>

      <td className="text-right">
        <UncontrolledDropdown>
          <DropdownToggle
            className="btn-icon-only text-light"
            href="#pablo"
            role="button"
            size="sm"
            color=""
            onClick={(e) => e.preventDefault()}
          >
            <i className="fas fa-ellipsis-v" />
          </DropdownToggle>
          <DropdownMenu className="dropdown-menu-arrow" right>
            <DropdownItem
              href="#pablo"
              onClick={(e) => e.preventDefault()}
            >
              Action
            </DropdownItem>
            <DropdownItem
              href="#pablo"
              onClick={(e) => e.preventDefault()}
            >
              Another action
            </DropdownItem>
            <DropdownItem
              href="#pablo"
              onClick={(e) => e.preventDefault()}
            >
              Something else here
            </DropdownItem>
          </DropdownMenu>
        </UncontrolledDropdown>
      </td>
    </tr>
  );
};

export default OrgRow;
