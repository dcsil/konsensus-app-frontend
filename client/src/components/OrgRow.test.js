import React from 'react';
import { shallow } from 'enzyme';
import OrgRow from './Profile/OrgRow';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

test('Test Sidebar component', () => {
  const wrapper = shallow(<OrgRow />);
  expect(wrapper).toMatchSnapshot();
});
