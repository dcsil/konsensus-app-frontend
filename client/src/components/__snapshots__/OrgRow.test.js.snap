// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test OrgRow component 1`] = `ShallowWrapper {}`;
